Assignment/dialogues.txt

```diff
@@ -36,7 +36,7 @@ V: People should not be afraid of their governments. Governments should be afrai
 
 Evey: Remember, remember, the Fifth of November, the Gunpowder Treason and Plot. I know of no reason why the Gunpowder Treason should ever be forgot... But what of the man? I know his name was Guy Fawkes and I know, in 1605, he attempted to blow up the Houses of Parliament. But who was he really? What was he like? We are told to remember the idea, not the man, because a man can fail. He can be caught, he can be killed and forgotten, but 400 years later, an idea can still change the world. I've witnessed first hand the power of ideas, I've seen people kill in the name of them, and die defending them... but you cannot kiss an idea, cannot touch it, or hold it... ideas do not bleed, they do not feel pain, they do not love... And it is not an idea that I miss, it is a man... A man that made me remember the Fifth of November. A man that I will never forget.
 
-V: V: Remember, remember, the Fifth of November, the Gunpowder Treason and Plot. I know of no reason why the Gunpowder Treason should ever be forgot.
+V: Remember, remember, the Fifth of November, the Gunpowder Treason and Plot. I know of no reason why the Gunpowder Treason should ever be forgot.
 
```

Comment:
Actually, this change is not made by others in other PRs, so maybe this can be a typo, but can you try without making a change in this file what say @sin-ha . Anyway, it's good for the first PR.

Reply:
It is a typo i guess this wasn't copied properly. Nice catch @sHiVaNgI821. I think it's better this way @Smit-create . This can be a serious PR now 🙂 if you wud make 1. Apart from gitingoring useless files